### PR TITLE
Update autobuild flags

### PR DESCRIPTION
**Pull request by plaindocs**
Recent release of sphinx-autobuild moved `-p` to `--port`. https://pypi.org/project/sphinx-autobuild/

I also removed the watch `_data` which never worked.

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
It would be useful to trigger a rebuild on the data dir, but this seems fine for now.

**Comment by plaindocs** [comment]
Agreed, I'll have another look at it.

File: docs/Makefile [hunk 1]
```diff
@@ -57,7 +57,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 livehtml:
-	sphinx-autobuild -p 8888  -z _data -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	sphinx-autobuild --port 8888 -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/html
```
Comment (plaindocs) [inline docs/Makefile]:
```suggestion
	sphinx-autobuild --port 8888 --watch _data -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

**Review by ericholscher — APPROVED** [review]
💯

**Pull request merged**